src/main/java/edu/cornell/kfs/fp/batch/service/impl/ProcurementCardSummaryFeedServiceImpl.java

```diff
@@ -196,18 +196,22 @@ protected void loadDataInDB(List<ProcurementCardSummaryEntry> entriesToLoad) {
 			try {
 				businessObjectService.save(entry);
 			} catch (RuntimeException e) {
-			    logClassLoaderDebugInfo();
+			    logClassLoaderDebugInfo(entry, retrievedEntry);
 			    throw e;
 			}
 		}
 	}
 	
-    private void logClassLoaderDebugInfo() {
+    private void logClassLoaderDebugInfo(ProcurementCardSummaryEntry entry, ProcurementCardSummaryEntry retrievedEntry) {
         Class repoClass = org.apache.ojb.broker.metadata.ClassDescriptor.class;
         LOG.info("Repository Class Loader: " + repoClass.getClassLoader());
 
         Class pcardSummaryEntryClass = edu.cornell.kfs.fp.businessobject.ProcurementCardSummaryEntry.class;
         LOG.info("ProcurementCardSummaryEntry Class Loader: " + pcardSummaryEntryClass.getClassLoader());
+        
+        LOG.info("ProcurementCardSummaryEntry Class Loader: entry attributes = {}", entry.toString());
+        
+        LOG.info("ProcurementCardSummaryEntry Class Loader: retrievedEntry attributes = {}", retrievedEntry.toString());

```

Comment:
`retrievedEntry` might be null at this point, so I would suggest adding some null checking here.